src/Messages/OWSIdentityManager.m

```diff
@@ -190,6 +192,14 @@ - (BOOL)saveRemoteIdentity:(NSData *)identityKey recipientId:(NSString *)recipie
             // Cancel any pending verification state sync messages for this recipient.
             [self clearSyncMessageForRecipientId:recipientId];
 
+            // TODO: This may be redundant with the "safety number changes"
+            //       messages.
```

Comment:
WRT a SN change, we should see this only and always when the previous identity was "verified" right (meaning the current one is "no longer verified")?

That seems OK, desirable even.